neutrinordp/xrdp-neutrinordp.c

```diff
@@ -182,19 +182,33 @@ lxrdp_connect(struct mod *mod)
         }
 
 #endif
-        LOG(LOG_LEVEL_ERROR, "freerdp_connect Failed to "
-            "destination :%s:%d",
+	mod->pid = g_getpid();
```

Comment:
Suggest removing this line and replacing `mod->pid` in the logging call with `g_getpid()`.

Reply:
Thank you.
I'll follow your suggestion.

Reply:
Modified by f33a381.